pkg/xds/generator/egress/testdata/input/02.internalservice-only.yaml

```diff
@@ -5,6 +5,8 @@ mtls:
   backends:
   - name: ca-1
     type: builtin
+routing:
+  zoneEgress: true
```

Comment:
because of wrong condition https://github.com/kumahq/kuma/pull/11445/files#r1763279132
I had to add it now to the test

It's not necessarily a bug in the real product. It seems that we were configuring configure egress with external services regardless of zoneEgress setting, which is fine. A bit excessive but should not have any significant impact